docs/sdk/dev/troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Troubleshooting
+description: Troubleshooting for the Spryker SDK document.
+template: concept-topic-template
+---
+# Troubleshooting
```

Comment:
please create a separate Troubleshooting section, like we have in SCOS (https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/troubleshooting-deployment-issues/troubleshooting-deployment-issues.html), Cloud, etc. projects. And add the file "spryker-sdk command not found" to this section. Also, please add this section and file to the sidebar